Tests/TABResourceLoaderTests/Helper functions/GlobalFunctionsTests.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  GlobalFunctionsTests.swift
+//  TABResourceLoader
+//
+//  Created by Luciano Marisi on 20/02/2017.
+//  Copyright © 2017 Luciano Marisi. All rights reserved.
+//
+
+import XCTest
+@testable import TABResourceLoader
+
+class GlobalFunctionsTests: XCTestCase {
+
+  func testMergingTwoDictionaryWithDifferentKeyValuePairs() {
+    let left = ["a_key": "a_value"]
+    let right = ["b_key": "b_value"]
+    guard let mergedDictionary = merge(left, right) else {
+      XCTFail(#function)
+      return
+    }
+    let expectedDictionary = ["a_key": "a_value", "b_key": "b_value"]
+    XCTAssertEqual(mergedDictionary, expectedDictionary)
+  }
+
+  func testMergingTwoNilDictionaryReturnsNil() {
+    XCTAssertNil(merge(nil, nil))
+  }
+
+  func testMergingLeftNilDictionaryWithRightNonNilDictionary() {
```

Comment:
minor thing but this function name looks the wrong way round to me, the left is non nil and right is nil, same as function below